### PR TITLE
Added Dino Run by DevKage

### PR DESCRIPTION
**Pull request by ufrshubham**

### CODE REVIEW

**Comment by spydon** [comment]
Hi, there is already a dino run in the list, is this one any different?

**Comment by ufrshubham** [comment]
@spydon if you are talking about the core game mechanics, then I would say both of them are similar.

**Comment by ufrshubham** [comment]
@spydon Does this mean I should not add my game here?

**Comment by spydon** [comment]
@ufrshubham we are trying to keep down the number of very similar games, there are a lot of people making the Dino game and the Langaw game for example. What do you think @flame-engine/flame-colab?

File: README.md [hunk 1]
```diff
@@ -68,6 +68,11 @@
 
 - [Maze generator](https://github.com/Dannark/FlameExamples) - List of algorithm examples to solve problems like Maze generation and more. By [Dannark](https://github.com/Dannark/)
 
+### YouTube videos
+
+<!--lint ignore double-link-->
+- [Dino Run - Flutter Game Development Series](https://youtube.com/playlist?list=PLiZZKL9HLmWOmQgYxWHuOHOWsUUlhCCOY) -  An easy to follow tutorial on how to make 2D mobile game in Flutter using Flame engine. By [DevKage](https://www.youtube.com/devkage)
```
Comment (spydon) [inline README.md:74]: This could be a dded under the "Other Articles & Tutorials" instead of under its own headline

Comment (ufrshubham) [reply README.md]: Made the change in latest commit.

**Comment by spydon** [comment]
@ufrshubham are you DevKage btw?

**Comment by ufrshubham** [comment]
Yes, I own the DevKage channel.

**Review by spydon — APPROVED** [review]

**Comment by StarTrooper08** [comment]
> Yes, I own the DevKage channel.

Great tutorial of Dino Run . 
Learned a lot about game development.

**Pull request merged**